text/0000-expand-math.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: Expand `math`
+- Start Date: 2022-02-28
+- RFC PR:
+- Pony Issue:
+
+# Summary
+
+This RFC proposes expanding the standard library's `math` package and the shape it might take to leverage the full extent of Pony. There was prior discussion on [Zulip](https://ponylang.zulipchat.com/#narrow/stream/192795-contribute-to.20Pony/topic/math.20lib) years ago, but I expect some opinions to have changed.
+
+# Motivation
+
+Currently, `math` includes limited functionality. This should be expanded to include math types, constants, and present a structure with further expansion in mind. An expanded `math` library will allow a unified mathematics among Pony developers.
+
+# Detailed design
+
+The primary goals of this initial expansion are: 
+
+1. restructure the `math` library into distinct sub-packages
+2. provide common `math` data types
+
+## Structure
+
+I propose a structure of distinct subpackages including the following:
+
++ `math/big`: Arbitrary precision numbers
++ `math/series`: Mathematical series
++ `math/constant`: Mathematical constants
```

Comment:
Small naming suggestion - `constants` feels more natural to me than `constant` as the name, in part because it matches the existing precedent of the `collections` package.

```suggestion
+ `math/constants`: Mathematical constants
```